framework/src/play/test/FunctionalTest.java

```diff
@@ -339,7 +340,23 @@ public InvocationContext getInvocationContext() {
     }
 
     public static Response makeRequest(final Request request) {
-        Response response = newResponse();
+        final Response response = newResponse();
+        response.onWriteChunk(new F.Action<Object>() {
+            public void invoke(Object chunk) {
+                byte[] bytes;
+                try {
+                    if ( chunk instanceof byte[]) {
+                        bytes = (byte[])chunk;
+                    } else {
+                        String message = chunk == null ? "" : chunk.toString();
+                        bytes = message.getBytes(response.encoding);
+                    }
+                    response.out.write(bytes);
+                } catch (IOException e) {
+                    e.printStackTrace();
```

Comment:
I don't know if we should log this (if the logger is available) or throw an exception that will be catch by the caller and will be wrapped into a 500